app/test3.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+def create_connection(db_url):
+    engine = create_engine(db_url)
+    base.metadata.bind = engine
```

Comment:
There's a typo in accessing the `metadata` attribute of the `Base` class. It should be `Base.metadata.bind = engine` instead of `base.metadata.bind = engine`.

**Label**: bug

**Code suggestion**

```python
Base.metadata.bind = engine
```